src/components/product.js

```diff
@@ -141,6 +143,14 @@ export default class Product extends Component {
       id = this.selectedImage.id;
       src = this.props.client.image.helpers.imageForSize(this.selectedImage, imageOptions);
       srcLarge = this.props.client.image.helpers.imageForSize(this.selectedImage, imageOptionsLarge);
+    } else if (this.selectedVariant.image == null && this.model.images[0] == null) {
+      id = null;
+      src = NO_IMG_URL;
+      srcLarge = NO_IMG_URL;
```

Comment:
You can use the `imageForSize` helper here too I think.

Reply:
I've tried putting this through `imageForSize` but I'm getting `403`s as a response with (the link that's generated)[https://sdks.shopifycdn.com/buy-button/latest/no-image_550x825.jpg]